docker/Dockerfile

```diff
@@ -12,6 +12,10 @@ ENV LD_LIBRARY_PATH=/usr/local/cuda/extras/CUPTI/lib64:${LD_LIBRARY_PATH} \
     CUDA_HOME=/usr/local/cuda-10.0 \
     PYENV_ROOT=/usr/local/pyenv
 
+# Add PPA for GCC 9
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
```

Comment:
IMHO: I think it is better to add the key as following commands without installing `software-properties-common` for `add-apt-repository` because `software-properties-common` will install some unnecessary packages. 
```
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 60C317803A41BA51845E371A1E9377A2BA9EF27F
echo 'deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu xenial main' | tee /etc/apt/sources.list.d/ubuntu-toolchain-r-ubuntu-test-xenial.list
```

For your reference, [Sample for Installing MongoDB[on Docker official doc]](http://docs.docker.jp/engine/examples/mongodb.html)

Reply:
@hadusam 
Thanks. Changed.